### PR TITLE
Added Hashicorp Packer

### PR DESCRIPTION
**Pull request by thechrisandrew**
Hi there! I added a badge for Packer from Hashicorp since i saw that there is Terraform on the list but not Packer. I have added it in the others section together with Terraform. Please let me know if there are mistakes I'll revise it immediately. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -693,6 +693,7 @@ You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
 | Jira          | ![Jira](https://img.shields.io/badge/jira-%230A0FFF.svg?style=for-the-badge&logo=jira&logoColor=white)                   | `![Jira](https://img.shields.io/badge/jira-%230A0FFF.svg?style=for-the-badge&logo=jira&logoColor=white)`                   |
 | Kubernetes    | ![Kubernetes](https://img.shields.io/badge/kubernetes-%23326ce5.svg?style=for-the-badge&logo=kubernetes&logoColor=white) | `![Kubernetes](https://img.shields.io/badge/kubernetes-%23326ce5.svg?style=for-the-badge&logo=kubernetes&logoColor=white)` |
 | Notion        | ![Notion](https://img.shields.io/badge/Notion-%23000000.svg?style=for-the-badge&logo=notion&logoColor=white)             | `![Notion](https://img.shields.io/badge/Notion-%23000000.svg?style=for-the-badge&logo=notion&logoColor=white)`             |
+| Packer     | ![Packer](https://img.shields.io/badge/packer-%2302A8EF.svg?style=for-the-badge&logo=packer&logoColor=white)    | `![Packer](https://img.shields.io/badge/packer-%2302A8EF.svg?style=for-the-badge&logo=packer&logoColor=white)`    |
```
Comment (kohasummons) [inline README.md:696]: Woohooo! @thechrisandrew, thanks for making this **contribution**🚀. I'm excited to have Packer join our humble collection of markdown badges. Recently, we've been looking at the issue of accessibility with our badges #355 making sure contributions going forward adhere to certain accessibility principles like contrast. 

That said, I've got a feeling we could tweak the contrast or the outlook of this badge a little to help with the case of accessibility. Here's what I came up with.

Packer

what do you think?

Comment (kohasummons) [reply README.md:696]: https://coolors.co/contrast-checker/ffffff-02a8ef

the contrast score of the initial badge

Comment (thechrisandrew) [reply README.md:696]: That looks awesome, quite tasteful too! One thing that kinda comes up to mind is if someone is actually using GitHub light mode, will the white background be a problem though? Other than that definitely works for me!!

Comment (kohasummons) [reply README.md:696]: Yikes! Thanks for catching that. That would have been a total disaster. After a few dystopian hours of head-scratching, I arrived at this. Better?

Packer

🤣 Just wondering why anyone would wanna use anything aside from dark mode

Comment (thechrisandrew) [reply README.md:696]: Eyy, that will definitely work for both light and dark theme discord users! I will update the code with your revisions in a moment.

image

> :rofl: Just wondering why anyone would wanna use anything aside from dark mode

I have a friend that uses light mode often, I honestly do not understand the motive behind it at all :laughing:

**Comment by thechrisandrew** [comment]
I have edited the badge color according to @kohasummons's reccomendation in the latest commit!

**Review by Ileriayo — APPROVED** [review]
lgtm

**Comment by kohasummons** [comment]
Glad we were able to do this @thechrisandrew. Can't wait to merge your next pull request 🔥

**Review by kohasummons — COMMENTED** [review]
:fire: :fire:

**Comment by thechrisandrew** [comment]
@kohasummons Thank you very much!

**Pull request merged**